agent/doc/swdesign/README.md

```diff
@@ -256,203 +317,240 @@ Needs:
+When the Ankaios Agent gets an add Workload command with the `UpdateWorklaod` message and the runtime of the Workload is unknown, the RuntimeManager shall skip this Workload.
 
 Tags:
-- AgentManager
+- RuntimeManager
 
 Needs:
 - impl
 - utest
 
-#### Agent Manager stores Workload to Runtime mapping
-`swdd~agent-manager-stores-workload-runtime-mapping~1`
+#### RuntimeManager stores Workload in the list of running workloads
+`swdd~agent-stores-running-workload~1`
 
 Status: approved
 
-When the Ankaios Agent gets an `UpdateWorkload` message with added Workloads and the runtime is known, the Agent shall store the mapping from workload name to runtime name.
+When the RuntimeManager creates new workload objects via the RuntimeFacade, the RuntimeManager shall store the Workload in a list of running workloads.
 
 Comment:
-The mapping is needed to be able to stop a workload as the runtime name is not part of the DeleteWorkload message.
+Please note that the object creation is targeted here and thus also resume or replace of running workloads is in scope.
+
+Rationale:
+The workload object is later used to update or delete the workload. The object also stores the ControlInterface for the workload and manages it during the lifetime of the workload.
 
 Tags:
-- AgentManager
-- ParameterStorage
+- RuntimeManager
 
 Needs:
 - impl
 - utest
 
-#### Handling the initial UpdateWorkload after Agent start (starting the RuntimeAdapters)
+#### Handling the initial UpdateWorkload after Agent start
 
 The following diagram and the subsequent requirements show the steps the Ankaios Agent takes when receiving the first UpdateWorkload command sent by Server. The first UpdateWorkload contains the complete initial list of workloads the Agent shall manage.
 
+
 ![Handling initial UpdateWorkload](plantuml/seq_update_workload_initial.svg)
 
-##### Agent Manager starts Runtime Adapters with initial list of workloads
-`swdd~agent-starts-runtimes-adapters-with-initial-workloads~1`
+##### RuntimeManager initial list of workloads handles existing workloads
+`swdd~agent-initial-list-existing-workloads~1`
 
 Status: approved
 
-After receiving the complete list of added workloads from the Ankaios Sever at the initial connection establishment, the Ankaios Agent shall start each RuntimeAdapter with the list of Workloads associated with this adapter.
+After receiving the complete list of added workloads from the Ankaios Sever at the initial connection establishment, the RuntimeManager shall handle existing workloads.
 
 Comment:
-In case the Agent was already running, the Runtime Adapter can take care of Workloads that were started in an earlier execution.
+In case the Agent was already running, the Runtime Adapter can take care of Workloads that were started in an earlier execution. Some of these workloads can be reused, some have to be updated and some stopped.
```

Comment:
I think the "Runtime Adapter" shall be replaced by the new term (the "RuntimeManager" inside this comment.

Reply:
```suggestion
In case the Agent was already running, the RuntimeManager can take care of Workloads that were started in an earlier execution. Some of these workloads can be reused, some have to be updated and some stopped.
```